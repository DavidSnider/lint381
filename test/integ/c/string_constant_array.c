const char foo[] = "bar";

const char* baz = "qux;"

char grault[] = "xyzzy";

const int plugh[] = {1, 2, 3};
