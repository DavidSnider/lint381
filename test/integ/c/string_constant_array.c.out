string_constant_array.c:1:15: error: Declare 'foo' as 'const char* const', not 'const char[]'
const char foo[] = "bar";
              ^^
